core/src/main/java/org/apache/gravitino/meta/DummyEntity.java

```diff
@@ -0,0 +1,154 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.gravitino.meta;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+import org.apache.gravitino.Entity;
+import org.apache.gravitino.Field;
+import org.apache.gravitino.HasIdentifier;
+
+/** A dummy entity that mainly used for temporary transactions or internal operations. */
+@EqualsAndHashCode
+@ToString
+public class DummyEntity implements Entity, HasIdentifier {
```

Comment:
The name `Dummy` is quite misleading, as far as I see, Files with the name `dummy` are mainly for testing and no real meaning.

Reply:
What you want is a class that can carry information such as name identifier, type, and meet the interface constraints.  

Both `InternalEntity` or `DummyEntity` are okay, since I also can't come up with a better name. By the way, you'd better add more comments about the functionality of class `DummyEntity`.

Reply:
more comments added

Reply:
How about `GenericEntity`?